<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>577</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>各种常见图表</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>数据</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableView" name="tableView">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>80</number>
           </attribute>
          </widget>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>300</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <attribute name="headerDefaultSectionSize">
            <number>90</number>
           </attribute>
           <attribute name="headerMinimumSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="headerShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>分数段</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>数学</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>语文</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>英语</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>平均分</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <item>
            <property name="text">
             <string>60分以下</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>60-69分</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>70-79分</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>80-89分</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>90分以上</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>平均分</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>最高分</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>最低分</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/3.ico</normaloff>:/images/images/3.ico</iconset>
        </attribute>
        <attribute name="title">
         <string>BarChart</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnBuildBarChart">
              <property name="text">
               <string>刷新柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="chartViewBar"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/43.ico</normaloff>:/images/images/43.ico</iconset>
        </attribute>
        <attribute name="title">
         <string>PieChart</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>分析数据</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cBoxCourse">
              <item>
               <property name="text">
                <string>数学</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>语文</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>英语</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>平均分</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDrawPieChart">
              <property name="text">
               <string>刷新饼图</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>HoleSize</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinHoleSize">
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>PieSize</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinPieSize">
              <property name="maximum">
               <double>1.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.700000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>主题</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cBoxTheme">
              <item>
               <property name="text">
                <string>Light</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BlueCerulean</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dark</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BrownSand</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BlueNcs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HighContrast</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BlueIcy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Qt</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>113</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="chartViewPie"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/281.GIF</normaloff>:/images/images/281.GIF</iconset>
        </attribute>
        <attribute name="title">
         <string>StackedBar</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnBuildStackedBar">
              <property name="text">
               <string>刷新堆叠柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>587</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="chartViewStackedBar"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/f4.ico</normaloff>:/images/images/f4.ico</iconset>
        </attribute>
        <attribute name="title">
         <string>PercentBar</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnPercentBar">
              <property name="text">
               <string>刷新百分比柱状图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>587</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="chartViewPercentBar"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/017.GIF</normaloff>:/images/images/017.GIF</iconset>
        </attribute>
        <attribute name="title">
         <string>ScatterChart</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnScatter">
              <property name="text">
               <string>刷新图表</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>646</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QChartView" name="chartViewScatter"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actGenData"/>
   <addaction name="actTongJi"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actTongJi">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/216.GIF</normaloff>:/images/images/216.GIF</iconset>
   </property>
   <property name="text">
    <string>数据统计</string>
   </property>
   <property name="toolTip">
    <string>数据统计</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actGenData">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/828.bmp</normaloff>:/images/images/828.bmp</iconset>
   </property>
   <property name="text">
    <string>初始化数据</string>
   </property>
   <property name="toolTip">
    <string>随机初始化数据</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
